e2etests/stable_test.go

```diff
@@ -1724,6 +1724,42 @@ code.height: 512
 package.height: 512
 `,
 		},
+		{
+			name: "crow_foot_arrowhead",
```

Comment:
can you add some styles on these too? e.g. stroke-width and stroke

Reply:
I've added styles on some of those, and I did introduce a slightly complicated connection. Is it what you expected?